static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -23,6 +24,7 @@ export const TESTS: Array<ABTest> = [
     new AcquisitionsEpicElectionInteractiveEnd(),
     new AcquisitionsEpicElectionInteractiveSlice(),
     new AcquisitionsThrasherUkElection(),
+    new EpicForBrexitCohort(),
```

Comment:
this test should be added to `acquisitions-test-selector` rather than here, otherwise there's a danger of two epics showing up together